apps/ngbank/src/app/app.config.ts

```diff
@@ -1,14 +1,61 @@
-import { ApplicationConfig } from '@angular/core';
+import {
+  ApplicationConfig,
+  importProvidersFrom,
+  isDevMode,
+} from '@angular/core';
 import {
   provideRouter,
   withEnabledBlockingInitialNavigation,
 } from '@angular/router';
 import { appRoutes } from './app.routes';
 import { provideAnimations } from '@angular/platform-browser/animations';
+import { provideState, provideStore } from '@ngrx/store';
+import {
+  createAccount$,
+  createJWTToken$,
+  login$,
+  logout$,
+  redirectAfterLogin$,
+  snackBarAfterError$,
+  userFeature,
+} from '@ngbank/user/store';
+import { provideEffects } from '@ngrx/effects';
+import { provideHttpClient } from '@angular/common/http';
+import { provideStoreDevtools } from '@ngrx/store-devtools';
+import {
+  MAT_SNACK_BAR_DEFAULT_OPTIONS,
+  MatSnackBarModule,
+} from '@angular/material/snack-bar';
 
 export const appConfig: ApplicationConfig = {
   providers: [
+    provideHttpClient(),
     provideRouter(appRoutes, withEnabledBlockingInitialNavigation()),
     provideAnimations(),
+    provideStore(),
```

Comment:
@santoshyadavdev would it be better to provide feature state in the `user.route.ts`?